asciidoc/modules/con_mg_changes-in-micrometer-metrics.adoc

```diff
@@ -0,0 +1,76 @@
+[id="changes-in-micrometer-metrics_{context}"]
+= Changes in Micrometer metrics
+
+The following section describes the changes in Micrometer metrics.
+
+== TCP sent and received bytes are recorded as counters with equivalent HTTP request and response summaries
+
+In prior releases, the following metrics were recorded as distribution summaries for sockets. From {VertX} {4} onward, these metrics are logged as counter, which report the amount of data exchanged.
+
+* Net client
++
+** `vertx_net_client_bytesReceived`
+** `vertx_net_client_bytesSent`
+
+* Net server
++
+** `vertx_net_server_bytesReceived`
+** `vertx_net_server_bytesSent`
+
```

Comment:
@vietj Are these metric names correct for Net and HTTP client and server?

Reply:
we should ask @jotak the question

Reply:
This is the old names, the new names are `*_bytes_read` / `*_bytes_written`.

Reply:
@jotak Thank you for the information. I have updated the metric names as below. Does the remaining note read ok?

**Net client**

- `vertx_net_client_bytes_read`

- `vertx_net_client_bytes_written`

**Net server**

- `vertx_net_server_bytes_read`

- `vertx_net_server_bytes_written`

Reply:
sounds good!

Reply:
Thank you so much for the review @jotak :)